src/CoreWCF.Primitives/src/CoreWCF/Security/SecurityTokenAttachmentMode.cs

```diff
@@ -0,0 +1,66 @@
+using System.ComponentModel;
+using System;
+namespace CoreWCF.Security
```

Comment:
Cleanup usings. Add blank line before namespace. Remove blank line before enum, Add internal accessor to helper method